src/components/AuthSection/AddCode.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+
+// Components
+import {
+  makeStyles,
+  IconButton,
+  Button,
+  TextField,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@material-ui/core';
+import { Add } from '@material-ui/icons';
+
+const AddCode = () => {
+  const classes = useStyles();
+  const [open, setOpen] = useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
```

Comment:
Standardize naming convention of function
Either call it `handlClose` and `handleOpen` 
or 
`handleClickOpen` and `handleClickClose`